### DISCUSSION TITLE
Option to hide read posts also hides them from user page

### DISCUSSION THREAD

**Discussion by Vyladence** [Docile-Alligator/Infinity-For-Reddit#1007] [Ideas]
In the "Post History" settings menu, you have an option to mark posts as read and hide them so you don't see the same post over and over when you refresh your feed. This part works well! 

Sometimes the scroll-past detection doesn't work fully but that could just be from me not scrolling far enough to trigger the post being read in my testing.

The problem arises when you're looking at a user profile or subreddit. If the post has been marked as read by the app, the post will be hidden completely. you won't be able to see the post if you view the profile of the person that posted it, nor if you view the subreddit it was posted in.

Ideally, the post would only be hidden from your main feed (and perhaps multireddits?), so you could still see the post if you view the profile or subreddit it was posted under.

**Comment by Wladefant** [comment]
Yes, or at least the pinned posts shouldn't hide

**Comment by rasc42** [comment]
I really agree that this behavior should change. It's really common for me to go back to a user's profile when I want to check if there has been any update on some discussions, or just to go through some old content from creators, so hiding posts there seems out of place to me.
